include/erlcloud_aws.hrl

```diff
@@ -42,7 +42,7 @@
           secret_access_key::string()|undefined|false,
           security_token=undefined::string()|undefined,
           %% epoch seconds when temporary credentials will expire
-          expiration :: pos_integer(),
+          expiration=undefined :: pos_integer()|undefined,
```

Comment:
i wonder why it's defaulted only here. is this necessary? 
i guess we default it to `undefined` everywhere or nowhere, agree?